pkg/system/rm.go

```diff
@@ -34,14 +35,24 @@ func EnsureRemoveAll(dir string) error {
 	}
 
 	for {
-		if err := resetFileFlags(dir); err != nil {
-			return fmt.Errorf("resetting file flags: %w", err)
-		}
 		err := os.RemoveAll(dir)
 		if err == nil {
 			return nil
 		}
 
+		// If the RemoveAll fails with a permission error, we
+		// may have immutable files so try to remove the
+		// immutable flag and redo the RemoveAll.
+		if errors.Is(err, syscall.EPERM) {
```

Comment:
You missed importing `syscall` package.

Reply:
Somehow this got lost in the rebase - should be fixed now